frontend/src/components/Buttons.css

```diff
@@ -0,0 +1,19 @@
+.buttons-container {
+    display: flex;
+    justify-content: space-around;
+    position: fixed;
+    bottom: 0;
+    width: 100%;
+    background-color: #0a0f80;
+    /* Set your desired background color */
+    padding: 10px;
+    box-shadow: 0px -2px 5px rgba(0, 0, 0, 0.1);
+    /* Optional: Add a subtle shadow */
+}
+
+.buttons-container button {
+    padding: 10px;
+    font-size: 16px;
+    cursor: pointer;
+    color: white;
+}
```

Comment:
For HTML/CSS/JS projects this is perfect. However, this project uses the Tailwind CSS library, which means this CSS goes directly into JSX.